website/client/src/components/payments/successModal.vue

```diff
@@ -87,13 +57,15 @@
         </template>
         <!-- give gift subscription (non-recurring)-->
         <template v-if="paymentData.paymentType === 'gift-subscription'">
-          <span
-            v-html="$t('paymentYouSentSubscription', {
-              name: paymentData.giftReceiver, months: paymentData.subscription.months})"
-          ></span>
+          <div>
```

Comment:
interesting--not requesting a change here, just curious for my own edification, what did adding this containing div help with?

Reply:
It helped it to display properly--IIRC, this was the part of the modal that wasn't ever displaying the text when someone gave a gift sub.